pkg/store/github/github.go

```diff
@@ -103,11 +104,15 @@ func (s *GitHubStore) ReloadCache(ctx context.Context) error {
 			return err
 		}
 
-		versions := make([]*core.ModuleVersion, len(tags))
-		for i, tag := range tags {
-			versions[i] = &core.ModuleVersion{
-				Version:   strings.TrimPrefix(tag.GetName(), "v"), // Terraform uses SemVer names without 'v' prefix
-				SourceURL: fmt.Sprintf("git::ssh://git@github.com/%s/%s.git?ref=%s", owner, name, tag.GetName()),
+		versions := make([]*core.ModuleVersion, 0)
+		for _, tag := range tags {
+			ver := strings.TrimPrefix(tag.GetName(), "v") // Terraform uses SemVer names without 'v' prefix
+			_, err := version.NewSemver(ver)
+			if err == nil {
```

Comment:
Denne kan inlines
```go
if _, err := version.NewSemver(ver); err != nil {
```

Reply:
Since we are checking that err is not nil, won't that make the code harder to read?